docs/markdown/Dependencies.md

```diff
@@ -43,6 +43,21 @@ You can pass the `opt_dep` variable to target construction functions
 whether the actual dependency was found or not. Meson will ignore
 non-found dependencies.
 
+Meson also allows to get variables that are defined in the
+`pkg-config` file. This can be done by using the
+`get_pkgconfig_variable` function.
+
+```meson
+zdep_prefix = zdep.get_pkgconfig_variable('prefix')
+```
+
+These variables can also be redefined by passing the `define_variable`
+parameter, which might be useful in certain situations:
+
+```meson
+zdep_prefix = zdep.get_pkgconfig_variable('prefix', define_variable: ['prefix', '/tmp'])
+```
+
```

Comment:
Wouldn't it make more sense to use a different example here, like getting libdir or includedir which is based on prefix, whilst redefining prefix?

Reply:
I also wonder if this shouldn't have been called `override_variable` instead, but too late now I guess :)

Reply:
Thanks for your review :)

> Wouldn't it make more sense to use a different example here, like getting libdir or includedir which is based on prefix, whilst redefining prefix?

I agree with you regarding the `prefix` variable. I've changed the retrieved variable to `libdir` so the effect of the `prefix` redefinition is more noticeable.

> I also wonder if this shouldn't have been called `override_variable` instead, but too late now I guess :)

I also agree with this. I would have named it `redefine_variable` (I also like your name), but the `pkg-config` option used for this is called [`define-variable`](https://linux.die.net/man/1/pkg-config). I ended using the same name just for consistency (which makes us wonder if there wouldn't be a better name for the option in `pkg-config` the first place).

Until 0.44 is released, where this option is presented, I suppose that it's not too late.